packages/build/src/log/messages/install.js

```diff
@@ -17,13 +17,13 @@ const logMissingPluginsWarning = function (logs, packages) {
   logArray(logs, packages, { color: THEME.errorSubHeader })
 }
 
-const logInstallLocalPluginsDeps = function (logs, localPluginsOptions) {
+const logInstallLocalDeps = function (logs, localPluginsOptions) {
```

Comment:
I think the longer name makes sense here, WDYT?

Reply:
:+1: